test-worker/test/asset-compute/worker/png-webp-resize/file.png

```diff
@@ -0,0 +1 @@
+../file.png
```

Comment:
Webp supports transparency. Could we add a test with transparency?
(For the matter, we should also add a test for transparency handling when doing png -> jpeg (curious to see what color jpeg will associate to the transparency) and png->gif too).

Reply:
We have some validation of transparency with the watermark tests. None for plain images here in the sdk.

I'll probably add more tests anyway and will compare with some test images. FYI, I am not merging this PR right now anyway.